contractcourt/chain_arbitrator.go

```diff
@@ -627,6 +613,20 @@ func (c *ChainArbitrator) Start() error {
 		}
 	}
 
+	// Subscribe to a single stream of block epoch notifications that we
+	// will dispatch to all active arbitrators.
+	blockEpoch, err := c.cfg.Notifier.RegisterBlockEpochNtfn(nil)
+	if err != nil {
+		return err
+	}
+
+	// Start our goroutine which will dispatch blocks to each arbitrator.
```

Comment:
This should wait for all arbs to start up before sending the notifications. Still need to look at how things are done on the other side, but if it doesn't fully wait for each on to start up, they may miss the notifications in the worst case.

Reply:
Moved this to after we have started up all of our chanarbs 👍